### PR TITLE
Update Configurations endpoints

### PR DESCRIPTION
**Pull request by gabrieltron**

### CODE REVIEW

**Comment by gabrieltron** [comment]
[[Config] Add new properties to postman](https://trello.com/c/trLAqOpu/1308-config-add-new-properties-to-postman)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1930,7 +1930,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": false,\n    \"operationsSummaryCreation\": false\n}"
+											"raw": "{\n    \"operationsImageCreation\": false,\n    \"geoimagesResolution\": 0.00001,\n    \"geoimagesShape\": \"SQUARE\",\n    \"geoimagesProjection\": \"EPSG:3857\",\n    \"geoimagesColorRamp\": {\n        \"0%\"  : [200,   0, 0],\n        \"35%\" : [255,  40, 0],\n        \"45%\" : [255, 150, 0],\n        \"55%\" : [255, 240, 0],\n        \"65%\" : [  0, 230, 0],\n        \"75%\" : [  0, 190, 0],\n        \"100%\": [  0, 130, 0],\n        \"nv\"  : [  0,   0, 0, 0]\n    },\n    \"fieldsAutoSync\": true,\n    \"fieldsAttachIntersection\": 10,\n    \"fieldsMergeIntersection\": 10\n}"
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1933]: Can we keep the value of the properties to the default value of 0.01?

File: Leaf API.postman_collection.json [hunk 2]
```diff
@@ -1967,7 +1967,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": false,\n    \"operationsSummaryCreation\": false\n}"
+											"raw": "{\n    \"operationsImageCreation\": false,\n    \"geoimagesResolution\": 0.00001,\n    \"geoimagesShape\": \"ROUND\",\n    \"geoimagesProjection\": \"EPSG:3857\",\n    \"geoimagesColorRamp\": {\n        \"0%\"  : [200,   0, 0],\n        \"35%\" : [255,  40, 0],\n        \"45%\" : [255, 150, 0],\n        \"55%\" : [255, 240, 0],\n        \"65%\" : [  0, 230, 0],\n        \"75%\" : [  0, 190, 0],\n        \"100%\": [  0, 130, 0],\n        \"nv\"  : [  0,   0, 0, 0]\n    },\n    \"fieldsAutoSync\": true,\n    \"fieldsAttachIntersection\": 10,\n    \"fieldsMergeIntersection\": 10\n}"
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1970]: Can we keep the value of the properties to the default value of 0.01?

File: Leaf API.postman_collection.json [hunk 3]
```diff
@@ -2067,7 +2067,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": true,\n    \"operationsSummaryCreation\": true,\n    \"fieldsAutoSync\" : true\n}"
+											"raw": "{\n    \"operationsImageCreation\": false,\n    \"geoimagesResolution\": 0.00001,\n    \"geoimagesShape\": \"SQUARE\",\n    \"geoimagesProjection\": \"EPSG:3857\",\n    \"geoimagesColorRamp\": {\n        \"0%\"  : [200,   0, 0],\n        \"35%\" : [255,  40, 0],\n        \"45%\" : [255, 150, 0],\n        \"55%\" : [255, 240, 0],\n        \"65%\" : [  0, 230, 0],\n        \"75%\" : [  0, 190, 0],\n        \"100%\": [  0, 130, 0],\n        \"nv\"  : [  0,   0, 0, 0]\n    },\n    \"fieldsAutoSync\": true,\n    \"fieldsAttachIntersection\": 10,\n    \"fieldsMergeIntersection\": 10\n}"
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:2070]: Can we keep the value of the properties to the default value of 0.01?

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (gabrieltron) [reply Leaf API.postman_collection.json]: Sure. Done in 336642efbc36d289772c3e8f712f4eedcfe53440

File: Leaf API.postman_collection.json (hunk 2 above)
Comment (gabrieltron) [reply Leaf API.postman_collection.json]: Done in 336642efbc36d289772c3e8f712f4eedcfe53440

File: Leaf API.postman_collection.json (hunk 3 above)
Comment (gabrieltron) [reply Leaf API.postman_collection.json]: Done in 336642efbc36d289772c3e8f712f4eedcfe53440

**Pull request merged**